rasa/nlu/tokenizers/tokenizer.py

```diff
@@ -0,0 +1,45 @@
+from typing import Text, List, Optional, Dict, Any
+
+from rasa.nlu.components import Component
+from rasa.nlu.constants import (
+    MESSAGE_RESPONSE_ATTRIBUTE,
+    MESSAGE_TEXT_ATTRIBUTE,
+    CLS_TOKEN,
+)
+
+
+class Token(object):
+    def __init__(self, text, offset, data=None):
+        self.offset = offset
+        self.text = text
+        self.end = offset + len(text)
+        self.data = data if data else {}
+
+    def set(self, prop, info):
+        self.data[prop] = info
+
+    def get(self, prop, default=None):
+        return self.data.get(prop, default)
+
+
+class Tokenizer(Component):
+    def __init__(self, component_config: Optional[Dict[Text, Any]] = None) -> None:
+        super(Tokenizer, self).__init__(component_config)
+
+        if "use_cls_token" in self.component_config:
+            self.use_cls_token = self.component_config["use_cls_token"]
+        else:
+            self.use_cls_token = True
```

Comment:
I would rather use `try-except` here and raise exception, to insure that there is no hidden behavior when people create custom tokenizers

Reply:
then we ensure, that it is part of defaults for all future subclasses